apps/opik-backend/src/main/java/com/comet/opik/domain/ExperimentDAO.java

```diff
@@ -485,7 +494,8 @@ private Publisher<Experiment> mapToDto(Result result) {
                 .feedbackScores(getFeedbackScores(row))
                 .traceCount(row.get("trace_count", Long.class))
                 .promptVersion(row.get("prompt_version_id", UUID.class) != null
-                        ? new Experiment.PromptVersion(row.get("prompt_version_id", UUID.class), null)
+                        ? new Experiment.PromptVersionLink(row.get("prompt_version_id", UUID.class), null,
```

Comment:
Nit: it was already happening, but better favour builders over constructors in the whole code base. It's harder to spot why a field is null.

Multiple occurrences of this in this PR.